wandb/sdk/wandb_run.py

```diff
@@ -3123,38 +3123,81 @@ def log_model(
                     - `s3://bucket/path`
-            model_name: (str, optional) An artifact name. String containing only the following alphanumeric characters: dashes, underscores, and dots.
+            name: (str, optional) An artifact name. String containing only the following alphanumeric characters: dashes, underscores, and dots.
                 This will default to the basename of the path prepended with the current
                 run id  if not specified.
             aliases: (list, optional) Aliases to apply to this artifact,
                     defaults to `["latest"]`
 
+        Examples:
+            ```python
+            run.log_model(
+                path="/local/directory",
+                name="my_model_artifact",
+                aliases=["production"],
+            )
+            ```
+
+            Invalid usage
+            ```python
+            run.log_model(
+                path="/local/directory",
+                name="my_entity/my_project/my_model_artifact",
+                aliases=["production"],
+            )
+            ```
+
+        Raises:
+            ValueError: if name has invalid special characters
+
         Returns:
             None
         """
         self._log_artifact(
-            artifact_or_path=path, name=model_name, type="model", aliases=aliases
+            artifact_or_path=path, name=name, type="model", aliases=aliases
         )
 
     @_run_decorator._noop_on_finish()
     @_run_decorator._attach
-    def use_model(self, model_name: str) -> FilePathStr:
+    def use_model(self, name: str) -> FilePathStr:
         """Download a logged model artifact.
 
         Arguments:
-            model_name: (str) A model artifact name.
+            name: (str) A model artifact name.
                 May be prefixed with entity/project/. Valid names
                 can be in the following forms:
-                    - name:version
-                    - name:alias
-                    - digest.
+                    - artifact_name:version
+                    - artifact_name:alias
+                    - artifact_name:digest.
+
+        Examples:
+            ```python
+            run.use_model(
+                name="my_model_artifact:latest",
+            )
+
+            run.use_model(
+                name="my_project/my_model_artifact:v0",
+            )
+
+            run.use_model(
+                name="my_entity/my_project/my_model_artifact:<digest>",
+            )
+            ```
+
+            Invalid usage
+            ```python
+            run.use_model(
+                name="my_entity/my_project/my_model_artifact",
+            )
+            ```
 
         Raises:
-            AssertionError: if type of artifact 'model_name' does not contain 'model'
+            AssertionError: if type of artifact 'name' does not contain 'model'
         Returns:
-            path: (StrPath) path to downloaded artifact file(s).
+            path: (str) path to downloaded artifact file(s).
```

Comment:
```suggestion
            path: (str) path to downloaded model file(s).
```
or model artifact, but we should rarely use artifact on its own in these docstrings (applies to other places in the docstrings where artifact stands alone)